docs/2.0. APIs.md

```diff
@@ -2,27 +2,28 @@
 
 _(c) AMWA 2017, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
 
-This document covers common aspects of Connection Management API. Many of these conventions are matched with the specification of NMOS IS-04. Common conventions may be refactored into a separate repository at a later date.
+This document covers common aspects of the Connection API.
 
 ## API Specifications
 
-The Connection Management API is specified using:
-*   The following sub-sections describing common API properties.
-*   [RAML](http://raml.org/) documents and [JSON schemas](http://tools.ietf.org/html/draft-zyp-json-schema-04) in the [APIs](../APIs/) folder.
+The Connection API is specified using:
+
+- The following sub-sections describing common API properties.
+- [RAML](http://raml.org/) documents and [JSON schemas](http://tools.ietf.org/html/draft-zyp-json-schema-04) in the [APIs](../APIs/) folder.
 
 Examples of JSON format output are provided in the [examples](../examples/) folder.
 
 ## API Validation
 
-JSON schemas are included with the RAML API definitions. These include validation for values used within the APIs. It is strongly recommended that implementers of a Connection Management API use these JSON schemas as part of a validation stage when receiving requests from API clients. Invalid requests should cause a 400 HTTP error (Bad Request) to be returned to the client.
+JSON schemas are included with the RAML API definitions. These include validation for values used within the APIs. It is RECOMMENDED that implementers of a Connection API use these JSON schemas as part of a validation stage when receiving requests from clients. Invalid resources SHOULD cause a `400` (Bad Request) HTTP error to be returned to the client.
```

Comment:
I'd switch 'resources' back to 'requests' here. I interpreted resources as something which the server owns, and should never cause a 400 to be returned.

Reply:
Yes I agree -- a 400 is always associated with a request so that makes sense.

Reply:
👍